basiclti/basiclti-api/src/java/org/sakaiproject/lti/api/LTIService.java

```diff
@@ -201,6 +201,15 @@
 	 * @return
 	 */
 	public String getToolLaunch(Map<String, Object> tool, String siteId);
+	
+	/**
+	 * 
+	 * @param siteId
+	 * @param filterId
+	 * @param exportType
+	 * @return
+	 */
+	public String getExportLaunch(String siteId, String filterId, int exportType);
```

Comment:
Method name isn't very clear, as this returns an export URL not a URL that starts an LTI launch. Something like `getExportUrl` would be much clearer.